pom.xml

```diff
@@ -1559,9 +1577,9 @@
     </license>
   </licenses>
   <scm>
-    <connection>scm:git:https://bitbucket.org/opencast-community/opencast.git</connection>
-    <developerConnection>scm:git:git@bitbucket.org:opencast-community/opencast.git</developerConnection>
-    <url>https://bitbucket.org/opencast-community</url>
+    <connection>scm:git:https://github.com/opencast/opencast.git</connection>
+    <developerConnection>scm:git:git@github.com:opencast/opencast.git</developerConnection>
```

Comment:
Since this requires a Github account I'm wondering if we should use https instead. No strong feelings though.

Reply:
are you referring to `connection` or `developerConnection`?

Reply:
IMO both.  Git is only useful once you're a committer.

Reply:
So I take this as a strongly suggestion to change to https for `developerConnection` too.

Reply:
> Git is only useful once you're a committer.

You can use it even if you are no committer. You just need a Github account. But if you are no committer, it's read-only. HTTPS has no requirements, that's why I thought it may be preferrable. In the end this doesn't matter much anyway I think.

Reply:
Using `git@` for the `developerConnection` I thought one most probably would have an github account. Well now there are both versions available in this pull request and it can be cherry-pick. In the end both yield the same result.